lib/smart_answer/calculators/self_assessment_penalties.rb

```diff
@@ -4,6 +4,7 @@ module SmartAnswer::Calculators
   class SelfAssessmentPenalties < OpenStruct
     ONLINE_FILING_DEADLINE_YEAR = SmartAnswer::YearRange.resetting_on("31 January").freeze
     ONLINE_FILING_DEADLINE_YEAR_FEB = SmartAnswer::YearRange.resetting_on("28 February").freeze
+    HMRC_C19_EXTENSION_PAYMENT_DEADLINE_YEAR = SmartAnswer::YearRange.resetting_on("1 March").freeze
```

Comment:
I _think_ this should be the 31st of March, rather than the 1st, as interest payments now kick in on the 1st of April.

Reply:
Bearing in mind that these dates denote the deadline, rather than the date the payment interest kicks in.

Reply:
👍  Thanks @1pretz1 - good spot - fixed